sdk/tests/conformance/attribs/gl-vertex-attrib-zero-issues.html

```diff
@@ -142,7 +156,65 @@
   gl.disableVertexAttribArray(3);
 }
 
+gl.disableVertexAttribArray(3);
+
+debug("");
+debug("Test that attaching an unallocated buffer for an enabled, unconsumed attrib 0 works and does not crash");
+// Tests for http://crbug.com/756293 (driver crash on macOS)
+var unallocatedBuffer = gl.createBuffer();
+var draws = [];
 
+draws.push(function() {
+  debug("* drawArrays");
+  gl.drawArrays(gl.TRIANGLES, 0, 3);
+});
+draws.push(function() {
+  debug("* drawElements");
+  gl.drawElements(gl.TRIANGLES, 60000, gl.UNSIGNED_SHORT, 0);
+});
+if (contextVersion >= 2) {
+  draws.push(function() {
+    debug("* drawArraysInstanced");
+    gl.drawArraysInstanced(gl.TRIANGLES, 0, 3, 1);
+  });
+  draws.push(function() {
+    debug("* drawElementsInstanced");
+    gl.drawElementsInstanced(gl.TRIANGLES, 60000, gl.UNSIGNED_SHORT, 0, 1);
+  });
+  draws.push(function() {
+    debug("* drawRangeElements");
+    gl.drawRangeElements(gl.TRIANGLES, 0, 60000, 60000, gl.UNSIGNED_SHORT, 0, 1);
+  });
+}
+
+var ext = gl.getExtension("ANGLE_instanced_arrays");
+if (ext) {
+  // These draw calls are expected to fail because the shaders do not consume
+  // any attribute with divisor 0 (because they do not consume any attributes).
+  // But they are added just in case.
+  draws.push(function() {
+    debug("* drawArraysInstancedANGLE");
+    ext.vertexAttribDivisorANGLE(0, 0);
+    ext.drawArraysInstancedANGLE(gl.TRIANGLES, 0, 3, 1);
+  });
+  draws.push(function() {
+    debug("* drawElementsInstancedANGLE");
+    ext.vertexAttribDivisorANGLE(0, 0);
+    ext.drawElementsInstancedANGLE(gl.TRIANGLES, 60000, gl.UNSIGNED_SHORT, 0, 1);
+  });
+}
+for (var i = 0; i < draws.length; ++i) {
+  var draw = draws[i];
+  gl.bindBuffer(gl.ARRAY_BUFFER, unallocatedBuffer);
+  gl.useProgram(pNoAttribs);
+  gl.enableVertexAttribArray(0);
+  gl.vertexAttribPointer(0, 3, gl.FLOAT, false, 0, 0);
+  draw();
+  // clear errors if any
+  gl.getError();
```

Comment:
Should we make sure there is no GL error? Otherwise you won't be able to catch a bug in your testing code.

If needed, you can move vertex attribs set up code into the draw functions.